blues/application/tasks.py

```diff
@@ -224,24 +224,54 @@ def generate_nginx_conf(role='www'):
         f.write(conf)
 
 
-def notify_deploy(role=None):
+def get_github_org():
+    from .project import git_repository
+
+    url = git_repository()
+    if 'url' not in url or not url.startswith('git@github.com:'):
+        return None
+
+    domain, repo = url.split(':')
+    org, repo = repo.split('/')
+    return org
+
+
+def notify_deploy(role=None, commits=None):
     from .project import project_name, git_repository_path
 
+    msg = '`{deployer}` deployed '
+    if role:
+        msg += '`{project}::{state}:{role}` '
```

Comment:
A single colon works, but it's a sort of separator, makes it simpler to see the difference IMO.

And I put the role last because the role isn't always provided, so it'll always be at the end that something either is or isn't there. And I think it makes sense with the role at the end, since states acts as environments, so `{project}::{state}:{role}` is roughly in the order of how "big" the scope of each part is.